central/postgres/schema/schema_test.go

```diff
@@ -0,0 +1,395 @@
+//go:build sql_integration
```

Comment:
So this test is useful for validation purposes, but just to confirm, you are eventually planning to remove the create table statements altogether, right?

Reply:
Yes, I will remove it later.